experimental/packages/opentelemetry-instrumentation-http/src/internal-types.ts

```diff
@@ -62,3 +62,13 @@ export const enum SemconvStability {
   /** Emit both stable and old semantic conventions*/
   DUPLICATE = 0x1 | 0x2,
 }
+
+/**
+ * Names of possible synthetic test sources.
+ */
+export const SYNTHETIC_TEST_NAMES = ['alwayson'];
+
+/**
+ * Names of possible synthetic bot sources.
+ */
+export const SYNTHETIC_BOT_NAMES = ['googlebot', 'bingbot'];
```

Comment:
I guess semconv punted on defining what user-agent strings should indicate 'test' or 'bot' synthetic time. Is "alwayson" a well-known user-agent string for testing?

Reply:
This is just a value used on the Azure side. Happy to have this list updated in further PRs, but this one provides a framework for adding these values (since like you mention it wasn't defined in sem conv).